### DISCUSSION TITLE
Visualizing topic data in namespace

### DISCUSSION THREAD

**Discussion by xiangyang-95** [foxglove/community#741] [Q&A]
I am using foxglove studio to visualize my ros2 message data. 
When I am using it to visualize the topic without namespace, it is working great.
However, when I am using it to visualize the data with topic namespace, the 3D panel is not working as it is not able to get the /tf data. (NOTE: the tf topic has been remap to /<robot-namespace>/tf as well)
May I know is there any ways to enable visualize the data in 3D panel with namespace topic?

**Comment by achim-k** [comment]
Can you confirm that the required transforms are available on the `/tf` topic? I assume that the frame names are namespaced as well? Could you share an example mcap/bag file which could be used to reproduce the problem?
